chain/jsonrpc/Cargo.toml

```diff
@@ -33,3 +32,10 @@ near-client = { path = "../client" }
 near-network = { path = "../network" }
 near-pool = { path = "../pool" }
 near-jsonrpc-client = { path = "client" }
+near-rpc-error-macro = { path = "../../tools/rpctypegen/macro" }
+
+[build-dependencies]
+_rand = { package = "rand", version = "0.6.5" }
```

Comment:
Why do you need this specific version of `rand`?

Reply:
It's a trick to make it compile. Take a look:

https://github.com/dalek-cryptography/ed25519-dalek/issues/89